src/javascript/app/pages/cashier/deposit_withdraw.js

```diff
@@ -110,32 +108,8 @@ const DepositWithdraw = (() => {
         return req;
     };
 
-    const getCashierURL = (bch_has_confirmed) => {
-        if (!/^BCH/.test(Client.get('currency')) || bch_has_confirmed || Client.get('cashier_confirmed')) {
```

Comment:
is `cashier_confirmed` removed from everywhere too?

Reply:
I forgot one place to set cashier_confirmed, let me fix it, thank you